rogii/version.cmake

```diff
@@ -0,0 +1,5 @@
+SET(ROGII_QT_MAJOR_VERSION 5)
```

Comment:
ROGII_MAJOR_VERSION, ROGII_MINOR_VERSION, ROGII_PATCH_VERSION and ROGII_PKG_VERSION for uniformity